sdks/csharp/fullClientApp_excerpt_model.cs

```diff
@@ -1,7 +1,7 @@
 Console.WriteLine();
 Console.WriteLine($"Upload a model");
-var typeList = new List<string>();
 string dtdl = File.ReadAllText("SampleModel.json");
-typeList.Add(dtdl);
+var models = new List<string>();
```

Comment:
You can instantiate the list along with the dtld like you did earlier in this PR